client/src/components/LogIn.tsx

```diff
@@ -27,16 +27,37 @@ const theme = createTheme();
 
 const LogIn = (props: HookProps) => {
   let navigate = useNavigate();
-  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [message, setMessage] = React.useState("");
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
-    const data = new FormData(event.currentTarget);
-    // Update the modal state 
-    props.setState(false);
-    // eslint-disable-next-line no-console
-    console.log({
-      email: data.get('email'),
-      password: data.get('password'),
-    });
+    try {
+      // set the login url
+      let res = await fetch("http://127.0.0.1:3000/login", {
```

Comment:
For the url, can you use the environment variable, like in: https://github.com/winterNebs/little-forests/blob/efb961ff7553eba3402e2e3f49c3c577842d21fa/client/src/components/Pages/BuilderPage.tsx#L125